articles/implementing-firebase-in-flutter/index.md

```diff
@@ -0,0 +1,258 @@
+Flutter is a popular platform when it comes to the development of cross-platform applications. Developers can target major platforms such as iOS, Android, Windows, and Linux with a single code base. Flutter reduces development time and associated costs. Furthermore, developers can use widgets to create a more flexible and interactive UIs. Incorporating Firebase in Flutter applications will enhance productivity significantly.
+
+### Introduction
+Firebase helps developers in creating high-quality applications. It provides functionalities such as real-time database, cloud storage, authentication, and hosting. This tutorial focuses more on the real-time database. Firebase offers synchronization that ensures that all connected devices are notified whenever data changes. In this tutorial, we will be building a simple expense tracker application. It will allow individuals to store and retrieve a list of expenses. Let&#39;s get started.
+
+### Goal of the Tutorial
+To implement Firebase in a Flutter application and use Firebase's Real-Time Database to store and retrieve data.
+
+### Prerequisites
+This tutorial is suitable for beginners.
+
+For you to follow along, you should have the [Flutter SDK](https://flutter.dev/docs/get-started/install) installed on your computer. For the IDE, you can either use [Android Studio](https://developer.android.com/) or [Visual Studio Code](https://code.visualstudio.com/download). You should have an [Firebase account](https://firebase.google.com/) to access its products.
+
+### Getting started
+Launch Android Studio and create a new Flutter project.
+
+![getting started](/engineering-educationimplementing-firebase-in-flutter/getting-started.png)
+
+Open the `pubspec.yaml` file and add the following Flutter dependencies.
+
+```
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  firebase_database: ^4.4.0
+  firebase_core : ^0.5.3
+```
+
+Use the `pub get` command to retrieve appropriate dependencies.
+
+Next, open your browser and navigate to the [Firebase](https://firebase.google.com/) website. We need to create a new firebase project, as shown below.
+
+![project name](/engineering-educationimplementing-firebase-in-flutter/project-name.png)
+
+Add your flutter application to Firebase by clicking on the android icon.
+
+![add app](/engineering-educationimplementing-firebase-in-flutter/add-app.png)
+
+Add your application's package name as shown in the image below. You can find your package name in the app-level – `build.gradle` file. Click register app after completing this step.
```

Comment:
Can you add a screenshot of the package name in the build.gradle file or instructions on where to find it in the file?